Update the labels of settings dropdown to specify moderators and fix some width related issues.

sahil839 opened the pull request:
<!-- What's this PR for?  (Just a link to an issue is fine.) -->
- Specify moderators in the options of various setting dropdowns.
- Two commits for increasing the width of dropdowns to have enough space for options even after translation.

Fixes #19562.
<!-- How have you tested? -->


<!-- If a UI change.  See:
  https://zulip.readthedocs.io/en/latest/tutorials/screenshot-and-gif-software.html
  -->


<!-- Also be sure to make clear, coherent commits:
  https://zulip.readthedocs.io/en/latest/contributing/version-control.html
  -->

sahil839 commented:
I have tested the width for german translation using Google translate as the new strings added were not being translated automatically by changing language setting.

andersk commented:
It’s crazy that we need to manually specify a width for these elements. They would automatically size themselves if not for Bootstrap’s `select { width: 220px }` rule. Why don’t we neutralize that with `select { width: auto }` so we can get rid of these hard-coded lengths? (We could use `min-width: 220px` if we’re worried about them getting too small, although if we want that it should go on an appropriately scoped rule, not a global `select`.)

timabbott commented:
@sahil839 can you post a screenshot?

sahil839 commented:
This is the screenshot before specifying Moderators in the option for the width css change.
In English (have included to cover the longest dropdown)
![Screenshot from 2021-09-01 18-18-08](https://user-images.githubusercontent.com/35494118/131674864-23c3a3bc-dbb5-49d5-a97e-9cc6c60069f2.png)
In German translation -
![Screenshot from 2021-09-01 18-17-47](https://user-images.githubusercontent.com/35494118/131674948-c2f9ca15-6feb-4f5a-9c94-549140da6ea1.png)

sahil839 commented:
Here is the one with updated options (new options are not being translated right now, I guess these need to be updated in transifex, so attaching only English language screenshot)-
![Screenshot from 2021-09-01 18-26-48](https://user-images.githubusercontent.com/35494118/131675285-44360d34-6a3f-4dbf-af51-d1f7782c2b1d.png)

sahil839 commented:
@timabbott have attached a few screenshots, do you need any other specific one?

timabbott commented:
Thanks!  I think the automatic width ends up making the page look pretty bad, in that the widths of the dropdowns end up being random (depending on exactly which options we offer).  The problem will probably go away once we add the user groups option, but @alya and I agree we should skip the change to automatic width given how it ends up looking, or at least split it to a separate PR where we can iterate on a `min-width` approach that avoids that visual effect.

sahil839 commented:
@timabbott I removed the auto width commit and instead increased the width from 250px to 325px to have enough space for German translation. Also added some commits which seems like a improvement but have addeed them as separate commits
in case we decide against it.

Now on the organization permissions page, there are two dropdowns which have different widths from others -
- one is bot creation policy dropdown which needs to be wider as it has a `generic bots` option.
- The restricted email domains is also wider. I tried making it same as the others. It is fine for the German translation but is not enough for other translations like Spanish. 

Apart from the permissions page, the realm message retention policy dropdown and user timezone setting dropdown also does not have enough width. For the realm message retention dropdown, the problem is for German translation and not for English.

sahil839 commented:
Attaching the screenshots where width of dropdowns have been modified. Have checked the width for translations and it is fine.

![Screenshot from 2021-09-02 14-47-33](https://user-images.githubusercontent.com/35494118/131818271-fed1fe70-3bfb-479a-a6d6-138655dd6a8e.png)
![Screenshot from 2021-09-02 14-47-35](https://user-images.githubusercontent.com/35494118/131818434-6e2e1b58-0811-40c5-ba1f-2e8d026c9ba0.png)
![Screenshot from 2021-09-02 14-47-56](https://user-images.githubusercontent.com/35494118/131818455-50d65e6d-fa47-4dd7-acd9-6f2a5ed5f8ca.png)

sahil839 commented:
@timabbott reminder for review.

timabbott commented:
Merged, thanks @sahil839!
